src/conn/conn_compact.c

```diff
@@ -500,6 +504,8 @@ __background_compact_find_next_uri(WT_SESSION_IMPL *session, WT_ITEM *uri, WT_IT
             WT_ERR(__background_compact_should_skip(session, key, id.val, &skip));
             if (!skip)
                 break;
+            else
+                WT_STAT_CONN_INCR(session, background_compact_skipped);
```

Comment:
You have the `break` before, no need for the `else`.
```suggestion
            WT_STAT_CONN_INCR(session, background_compact_skipped);
```

Reply:
TY for the suggestion :) 70922ff7bd118b38684c9e3f3999688408f9c4c3